Maskbook/Wallet/WalletConnect/Client/WalletConnectClient+Supported.swift

```diff
@@ -34,16 +34,12 @@ extension WalletConnectClient {
     }
 
     static func openWalletIfInstalled(installedWallet: InstalledWallet) {
-        if installedWallet.name == "Rainbow" {
+        if installedWallet.scheme.isNotEmpty {
             UIApplication.shared.open(URL(string: installedWallet.scheme)!)
```

Comment:
```
if canOpen(scheme) {
    open
    return
}
```

Reply:
When the scheme fails to open, we could have a fall back solution

Reply:
I think fall back solution is doing nothing, because the app has been deleted, universal can't open app too.

Reply:
<img width="712" alt="Screen Shot 2022-05-11 at 16 59 06" src="https://user-images.githubusercontent.com/6863518/167811620-2d79eb91-083e-4555-8c86-eda693d844e8.png">
`if canOpen(scheme) {
    open
    return
}`

This kind of logic added in picture above